hail_search/queries/mito.py

```diff
@@ -325,6 +326,15 @@ def _add_project_lookup_data(self, ht, annotation_fields, *args, **kwargs):
                 hl.dict(family_indices.map(lambda j: (lookup_ht.project_families[project_guid][j], True))),
             ))), 1),
         )[0]
+
+        for project_guid, families in variant_projects.items():
+            if hfs.exists(self._get_table_path(f'projects/WES/{project_guid}.ht', use_ssd_dir=True)):
```

Comment:
Is there a need for `hfs` here?  `hfs` is for cloud-agnosticism but I don't think we'll ever be serving the project tables from cloud (much slower!). `os.path.exists` is probably fine?

Just for kicks:

```
In [3]: %timeit os.path.exists('a_file')
983 ns ± 10.8 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)

In [4]: %timeit hfs.exists('a_file')
194 µs ± 566 ns per loop (mean ± std. dev. of 7 runs, 1,000 loops each)
```